esx_service/vmdk_ops.py

```diff
@@ -969,12 +969,7 @@ def connectLocalSi(force=False):
                 version=newestVersions.Get('vim'))
         except Exception as e:
             logging.exception("Failed to create the local Service Instance as 'dcui', continuing... : ")
-    elif force:
-        logging.warning("Reconnecting to the local Service Instance")
-        _service_instance = pyVim.connect.Connect(
-            host='localhost',
-            user='dcui',
-            version=newestVersions.Get('vim'))
```

Comment:
This call is exactly the same as on L966 but without try catch block. Exception can occur here too. Removing the force flag and resetting _service_instance and calling connectLocalSi for cleaner connect.